app/assets/v2/js/pages/process_bounty.js

```diff
@@ -1,6 +1,17 @@
 /* eslint-disable no-console */
 window.onload = function() {
 
+  const changeRateUser = () => {
```

Comment:
could we rename this to rateContributor / addRating to make it more meaningful  ?